North-Seattle-College/ad440-winter2021-thursday-repo
# Capture average time to load /users/{userId} with cache

Closes #308 

This PR uses Puppeteer to load the /users/2 page and time it.
It does this 40 times, taking & saving screenshots, and averages the time.

My results: 
Total time: 21.789999999999996 seconds
Average time: 0.5447499999999998 seconds

**To test:**
- fork my repo & checkout the branch yaholl/sprint6
- navigate to /test/ui
- in terminal, run `node time-users-id.js`
- you should see confirmation of 40 screenshots being taken, along with the time it took to load the page each time
- at the end,  you will see the Total time and Average time
- you can look in test/results/ui-test-results/users-id
  - you should see a new folder called `group-` and the current date/time
  - inside, there should be 40 screenshots of the page
